### ISSUE TITLE
Python 2 Expression tab

### ISSUE THREAD

**Issue by matiasmorant** [wvlia5/SublimeDebugger#11]
Expressions are all evaluated to None

**Comment by matiasmorant** [comment]
```
Traceback (most recent call last):
  File "/home/matias/.config/sublime-text-3/Packages/SublimeDebugger/backends/comm_utils.py", line 69, in __getitem__
    ret = eval("self."+instruction)(*json.loads(parameters))
  File "backends/dbpy2_server.py", line 24, in D_tryeval
    return DB.tryeval(expr)
NameError: global name 'DB' is not defined
```

_globals_ are being overwritten to the globals of the debugged file.
Maybe it has to do with the black magic happening in _runscript_ which overwrites  \_\_main__.\_\_dict__, because expressions evaluate "correctly" before _runscript_ is called.

**Comment by matiasmorant** [comment]
Solved ba2edf2ab876f2d2acc85cea753080f5a01e8392
